exporter/clickhousetracesexporter/clickhouse_exporter_v3.go

```diff
@@ -38,11 +38,19 @@ func populateCustomAttrsAndAttrs(attributes pcommon.Map, span *SpanV3) {
 			}
 			span.ExternalHttpUrl = value
 			span.HttpUrl = v.Str()
+		} else if k == "http.target" || k == "url.path" {
+			span.UrlPath = v.Str()
+		} else if k == "net.peer.name" || k == "server.address" {
```

Comment:
logic is duplicate and `server.address` is conflicting in `span.ServerAddress` and `span.HttpHost`